spring-kafka/src/main/java/org/springframework/kafka/annotation/KafkaListenerAnnotationBeanPostProcessor.java

```diff
@@ -90,6 +91,8 @@
 import org.springframework.util.StringUtils;
 import org.springframework.validation.Validator;
 
+import reactor.core.publisher.Flux;
```

Comment:
With this Reactor becomes as not a `optional` dependency any more...
Is it really what we are aiming now ?